pkg/policy/testing/helper.go

```diff
@@ -0,0 +1,148 @@
+package testing
+
+import (
+	"context"
+	"encoding/csv"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+	"path/filepath"
+	"sort"
+	"strings"
+	"testing"
+
+	sdkdb "github.com/cloudquery/cq-provider-sdk/database"
+	"github.com/google/go-cmp/cmp"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/cloudquery/cloudquery/pkg/policy"
+	"github.com/cloudquery/cq-provider-sdk/testlog"
+	"github.com/hashicorp/go-hclog"
+)
+
+// pol policy.Policy
+func TestPolicy(t *testing.T, source, selector, snapshotDirectory string) {
+	// var pols policy.Policies
+	// pols = append(pols, &policy.Policy{Name: "aws", Source: source})
+	// Setup dependencies
+	t.Helper()
+	uniqueTempDir, err := os.MkdirTemp(os.TempDir(), "*-myOptionalSuffix")
+	if err != nil {
+		t.Fatal(err)
+	}
+	l := testlog.New(t)
+	l.SetLevel(hclog.Info)
+
+	ctx := context.Background()
+	conn, err := sdkdb.New(ctx, hclog.NewNullLogger(), "postgres://postgres:pass@localhost:5432/postgres")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	policyManager := policy.NewManager(uniqueTempDir, conn, l)
+	p, err := policyManager.Load(ctx, &policy.Policy{Name: "test-policy", Source: source})
+	if err != nil {
+		t.Fatal(err)
+	}
+	pol := p.Filter(selector)
+
+	clie := policy.NewExecutor(conn, l, nil)
```

Comment:
nit: weird name for a policy executor

Reply:
Fair comment, will fix. This is an old name when I had a different executor that was cli executor